apps/mobile/src/features/browser/approver-sheet/stx/call-contract/call-contract.tsx

```diff
@@ -7,7 +7,7 @@ import { useBroadcastStxTransaction } from '@/queries/stacks/use-broadcast-stx-t
 import { useAccounts } from '@/store/accounts/accounts.read';
 import { App } from '@/store/apps/utils';
 import { useStacksSigners } from '@/store/keychains/stacks/stacks-keychains.read';
-import { assertStacksSigner } from '@/store/keychains/stacks/utils';
+import { assertReadWriteStacksSigner } from '@/store/keychains/stacks/utils';
```

Comment:
Maybe we could instead keep this as `assertStacksSigner` and assert `if (!signer || !signer.isReadonly)` instead?